### ISSUE TITLE
Allow separable calibration smoothing between redcal and abscal

### ISSUE THREAD

**Issue by jsdillon** [HERA-Team/hera_cal#559]
As @AaronParsons argued on today's analysis telecon, once #537 is done, the blacklisting of LSTs due to a bad abscal model will throw away information from redcal as well. It also mandates that we smooth using a timescale appropriately long to bridge the gaps in LST.

In theory, if we rephase redcal to a reference triad (one antenna for global phase and amplitude, another two to set the tip and tilts), we can separate redcal and abscal without discontinuities, smooth them both however we'd like (and in parallel) and then recombine them together at the end. This isn't strictly necessary for H3C, but it's a nice-to-have.

Let's revisit when #537 is done and smooth_cal is integrated into RTP.
